core/primer.md

```diff
@@ -118,3 +118,37 @@ points wother noting:
   been deleted scucessfully. It can be used even if the current "latest"
   Version isn't being deleted.  Note that a `404` in the `DELETE .../ID` case
   is an error and no changes to the "latest" Version will occur.
+
+# Latest Version and Maximum Versions
+
+Each Resource type can specific the maximum number of Versions that the
+server must save. Once that limit is reached then it must delete Verisons
+to stay within the limit - by deleting oldest Versions first. However, since
+tagging a Version as "latest" marks that Version as special, this pruning
+logic must skip the "latest" Version. There is one exception to this rule.
+If the maximum Versions is set to 1 then when a new Version is created, that
+Verison will become the "latest" Version regardless of whether or not the
+user asked for it to be.
+
+In general, during an operation that creates, updates or deletes the Versions
+of a Resource, the following logic is applied:
+- Modify the Versions collection as requested
+- Apply the "latest" processing logic by setting (or not) which Version is the
+  "latest"
+- If the number of Versions exceeds the maximum allowed Verisons then, starting
+  with the oldest, keep deleting until the collection is within the limit.
+  Except if the limit is 1, in which case if a new Version is created then it
+  it tagged as "latest"
+
+Let's walk through a complex example:
+- Max allowed Versions is 2
+- Initially the following Versions exist: v4, v2 (latest)
+- Max lloaed Versions is now set to 0
```

Comment:
What does it mean to set the maximum allowed Versions to 0? I suspect it's intended to mean "unlimited" but I can't see that specified (at least in the change).

Reply:
I added some text here to say that 0=unlimited, to repeat what the spec says.